src/app/shared/api.service.ts

```diff
@@ -11,30 +12,48 @@ export class ApiService {
 
 	private apiURL: string = "http://localhost:3000/task";
 
-	public constructor(private http: HttpClient) { }
+	public constructor(
+		private http: HttpClient,
+		private localStorageService: LocalStorageService
+	) { }
 
 	public getAllTasks(): Observable<TaskDTO[]> {
 		const getAllTasksUrl = this.apiURL + "/all";
-		return this.http.get<TaskDTO[]>(getAllTasksUrl);
+		const token = this.localStorageService.getToken();
+		return this.http.get<TaskDTO[]>(getAllTasksUrl, {
+			headers: { "Authorization": "Bearer " + token, "Content-Type": "application/json" }
+		});
```

Comment:
declare an object in order to store the headers and pass that object to the methods